src/System.Memory/src/System.Memory.csproj

```diff
@@ -27,12 +24,22 @@
     <Compile Include="System\SpanHelpers.Clear.cs" />
     <Compile Include="System\ThrowHelper.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
>@joperezr, is there any reason to build for older frameworks anymore?

To answer your question, yes, if the library is an OOB library (e.g. shipping in its own package, which System.Memory is one of those cases) then we do want to keep old configurations. If it will ship as part of our netcoreapp flat package, then it's ok to remove old configurations